File: test/ayush.dev/python/subgraphs/SG_LIVY/code/org/python/SG_LIVY/SG_LIVY_1/Reformat_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pythonlivy.config.ConfigStore import *
from pythonlivy.udfs.UDFs import *

def Reformat_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: test/ayush.dev/scala/pipelines/SclalaLivy/code/src/main/scala/io/prophecy/pipelines/livy_scala/graph/Subgraph_1/Reformat_2.scala
package io.prophecy.pipelines.livy_scala.graph.Subgraph_1

import io.prophecy.libs._
import io.prophecy.pipelines.livy_scala.config.ConfigStore._
import io.prophecy.pipelines.livy_scala.udfs.UDFs._
import io.prophecy.pipelines.livy_scala.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_2 {
  def apply(spark: SparkSession, in: DataFrame): DataFrame = in
}
